### PR TITLE
Initial Work on People Page

### PR DESCRIPTION
**Pull request by shahnozayadgar**
- [ ] - Include all members with images, names, email address, MS thesis (PhD students), Universities (interns+visiting researchers) 
- [ ] Include Alumni and Special thanks 
Screenshot 2024-02-29 at 3 20 44 PM

### CODE REVIEW

**Review by alextio — APPROVED** [review]

File: src/data/members.ts [hunk 1]
```diff
@@ -1,49 +1,297 @@
...
+    firstName: 'Yoonah',
+    lastName: 'Park',
+    email: 'wisdomsword21@snu.ac.kr',
+    career: 'Undergraduate Intern',
+    img: 'yoonah.jpg',
+  },
+  shahnozayadgarova: {
+    firstName: 'Shahnoza',
+    lastName: 'Yadgarova',
+    img: 'noza.jpeg',
```
Comment (alextio) [inline src/data/members.ts:276]: Missing email, career field here

**Review by alextio — CHANGES_REQUESTED** [review]
Noza's field in members.ts is missing email and career field causing errors in checks. Please add those.

**Comment by jhw123** [comment]
Could you fix the type error first?

**Comment by alextio** [comment]
Too many file changes. Best to restart this PR with a clean branch.

**Pull request closed without merging**